src/main/java/picard/sam/PositionBasedDownsampleSam.java

```diff
@@ -98,24 +159,36 @@
     @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "The input SAM or BAM file to downsample.")
     public File INPUT;
 
-    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output, downsampled, SAM or BAM file to write.")
+    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "The output, downsampled, SAM or BAM file.")
     public File OUTPUT;
 
-    @Argument(shortName = "F", doc = "The (approximate) fraction of reads to be kept, between 0 and 1.", optional = false)
+    @Argument(shortName = "F", doc = "The (approximate) fraction of reads to be kept, between 0 and 1.")
     public Double FRACTION = null;
 
+    @Argument(doc = "Determines whether the duplicate tag should be reset since the downsampling requires re-marking duplicates.")
+    public boolean REMOVE_DUPLICATE_INFORMATION = true;
+
+    @Argument(doc = "Regular expression that can be used to parse read names in the incoming SAM file. Read names are " +
+            "parsed to extract three variables: tile/region, x coordinate and y coordinate. The x and y coordinates are used " +
+            "to determine the downsample decision. " +
+            "Set this option to null to disable optical duplicate detection, e.g. for RNA-seq " +
+            "The regular expression should contain three capture groups for the three variables, in order. " +
+            "It must match the entire read name. " +
+            "Note that if the default regex is specified, a regex match is not actually done, but instead the read name " +
+            " is split on colon character. " +
+            "For 5 element names, the 3rd, 4th and 5th elements are assumed to be tile, x and y values. " +
+            "For 7 element names (CASAVA 1.8), the 5th, 6th, and 7th elements are assumed to be tile, x and y values.")
+    public String READ_NAME_REGEX = ReadNameParser.DEFAULT_READ_NAME_REGEX;
```

Comment:
The default value isn't rendering for me in `picardDoc` (which is extra inconvenient for a regex).

Reply:
"that can be"?

Howzabout "Use these regular expressions to parse read names in the input SAM file" or sumpin?

Moving on now ...

Reply:
@danxmoran I see this:

```
READ_NAME_REGEX=String        Use these regular expressions to parse read names in the input SAM file. Read names are
                              parsed to extract three variables: tile/region, x coordinate and y coordinate. The x and y
                              coordinates are used to determine the downsample decision. Set this option to null to
                              disable optical duplicate detection, e.g. for RNA-seq The regular expression should
                              contain three capture groups for the three variables, in order. It must match the entire
                              read name. Note that if the default regex is specified, a regex match is not actually
                              done, but instead the read name is split on colons (:). For 5 element names, the 3rd, 4th
                              and 5th elements are assumed to be tile, x and y values. For 7 element names (CASAVA 1.8),
                              the 5th, 6th, and 7th elements are assumed to be tile, x and y values.  Default value:
                              <optimized capture of last three ':' separated fields as numeric values>. This option can
                              be set to 'null' to clear the default value. 
```

note the "<optimized capture of last three ':' separated fields as numeric values>" part

Reply:
Ah, I was talking about the HTML:
![screen shot 2018-01-05 at 5 10 02 pm](https://user-images.githubusercontent.com/2755881/34630712-51da60f2-f23b-11e7-9975-2dc4aefcdb71.png)
Was expecting the default to render into that middle column somehow.